src/basics.md

```diff
@@ -194,23 +184,30 @@ fn main() {
 
 [ex-rand-dist]: #ex-rand-dist
 <a name="ex-rand-dist"></a>
-## Generate random numbers with normal distribution
+## Random number distributions
 
 [![rand-badge]][rand] [![cat-science-badge]][cat-science]
 
-Creates a [`Normal`] distribution with mean `3` and standard deviation `5`
-and generates a random value with [`IndependentSample::ind_sample`].
+You've already seen how to create numbers with uniform distribution [above][ex-rand-range]
```

Comment:
we don't aim to make the cookbook into a tutorial (what I've meant by "progression" was if there is any missing information between examples or if these are not sorted by their difficulty). We strive for problem solution (hence cookbook) approach of self-contained examples. 

I would roll back the original description here as it was short and to the point and just link to alternative distributions.